app/config/config.yml

```diff
@@ -61,3 +61,8 @@ fos_rest:
         rules:
             - { path: '^/api/.*', priorities: ['json', 'xml'], fallback_format: json, prefer_extension: true }
             - { path: '^/', stop: true }
+
+twig:
```

Comment:
Can we add it to `@CoreBundle/Resources/config/app/config.yml` instead? Btw. we can name it `sylius_meta`.

Reply:
👍

Reply:
done.